tgui/packages/tgui/interfaces/MusicInstrument.tsx

```diff
@@ -0,0 +1,183 @@
+import { useBackend, useLocalState } from '../backend';
+import { Button, Flex, LabeledList, Box, Knob, Input } from '../components';
+import { Window } from '../layouts';
+
+type MusicInstrumentData = {
+  name: string;
+  notes: string[];
+};
+
+export const MusicInstrument = (props, context) => {
```

Comment:
```suggestion
export const MusicInstrument = (_props, context) => {
```
Helps various things know this is not expected to be used.